pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>com.djrapitops</groupId>
     <artifactId>Extension-GriefDefender</artifactId>
-    <version>R0.2</version>
+    <version>1.0-SNAPSHOT</version>
```

Comment:
Please keep R___ in the version. Generally the version for extensions is `<version of the plugin's API>-R<version of extension>`